Python/Include/Scalar.hpp

```diff
@@ -148,18 +149,19 @@ namespace Rover {
   PythonScalar<T>::operator T&() {
     return m_value;
   }
-
-  template<typename T>
-  PythonScalar<T> abs(PythonScalar<T> scalar) {
-    return ::abs(static_cast<T>(scalar));
-  }
 }
 
 namespace pybind11 {
   template<typename T>
   object cast(const Rover::PythonScalar<T>& scalar) {
     return cast(static_cast<T>(scalar));
   }
+
+  template<typename T>
```

Comment:
Can you move this back to the Rover namespace? I can see some circumstances where we absolutely must add functions to a third party library, but it seems arbitrary to put this in the ```pybind``` namespace.

Reply:
It only tries to resolve to `Rover::abs` if I do it in `namespace Rover`. Not sure whether it is standard or a VS issue. It can be in any other namespace, those nested in `namespace Rover` included. I can put it into `namespace Rover::Details` if appropriate.

Reply:
Fair enough, put it in ```Details``` and also include ```<cmath>```.